lib/ansible/cli/doc.py

```diff
@@ -1144,6 +1144,7 @@ def get_role_man_text(self, role, role_json):
                 text.append("%s\n" % textwrap.fill(DocCLI.tty_ify(desc),
                                                    limit, initial_indent=opt_indent,
                                                    subsequent_indent=opt_indent))
+
```

Comment:
Better also remove this new empty line :)

Reply:
Sorry, I missed that one.- It's gone now.